keyboards/miiiw/blackio83/rev_0100/rules.mk

```diff
@@ -0,0 +1,29 @@
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = yes       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+NKRO_ENABLE = yes           # Enable N-Key Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+RGB_MATRIX_ENABLE = yes
+RGB_MATRIX_DRIVER = custom
+DIP_SWITCH_ENABLE = yes     # Enable dip switches
+DEBOUNCE_TYPE = sym_defer_pk    # Debounce using eager_pk.
+CUSTOM_MATRIX = lite
+DEFERRED_EXEC_ENABLE = yes
+NO_USB_STARTUP_CHECK = yes
+# EEPROM_DRIVER = i2c
```

Comment:
All of this can be moved to the info.json.   `qmk info -kb (name) -f ascii` can help with that